src/Twig/Extension/NumberExtension.php

```diff
@@ -37,7 +37,7 @@ public function getFilters(): array
         return [
             new TwigFilter('toReadableSize', 'Cake\I18n\Number::toReadableSize'),
             new TwigFilter('toPercentage', 'Cake\I18n\Number::toPercentage'),
-            new TwigFilter('number_format', 'Cake\I18n\Number::format'),
+            new TwigFilter('cake_number_format', 'Cake\I18n\Number::format'),
```

Comment:
Why are others camelCase but these underscored?

Reply:
We should follow whatever convention Twig uses. I believe it uses underscored names for methods/filters. We could deprecate the current camelBacked ones and add their underscored versions